pages/about_data_page.py

```diff
@@ -0,0 +1,60 @@
+accessible through the [UK Data Service SecureLab](https://ukdataservice.ac.uk/) by accredited researchers.
+                """
+    )
+    st.markdown(
+        """The SERL data is a rich source of information, including half-hourly gas and electricity consumption data and household survey data, such as property type, household composition, and income. The data documentation is available on the [data catalogue](https://beta.ukdataservice.ac.uk/datacatalogue/studies/study?id=8666#!/documentation).
+                """
+    )
+    st.markdown(
+        """You can find detailed information about the underlying methodology in the [appendix]().
+                """
+    )
+
+    st.markdown("## Data citation")
+    st.markdown(
+        """
+Elam, S., Few, J., McKenna, E., Hanmer, C., Pullinger, M., Zapata-Webborn, E., Oreszczyn, T., Anderson, B., Department for Levelling Up, Housing and Communities, European Centre for Medium-Range Weather Forecasts, Royal Mail Group Limited. (2024). Smart Energy Research Lab Observatory Data, 2019-2024: Secure Access. [data collection]. 8th Edition. UK Data Service. SN: 8666, DOI: http://doi.org/10.5255/UKDA-SN-8666-8
+
+                """
+    )
+
+    st.markdown("## Caveats and limitations of the work")
+    with st.expander("Small sample of households & information sharing"):
+        st.markdown(
+            """The Smart Energy Research Lab (SERL) data contains smart meter and contextual data from over 13,000 households. However, not all of these households met the criteria for inclusion in this analysis. For instance, some were excluded because their data did not span the entire analysis period. Ultimately, to ensure the sample remains representative of Great Britain, we focused our analysis on a smaller, refined sample of approximately 6,000 households.
+The process of defining the number of energy profiles is a careful balance of data science insights and expert domain knowledge. It's crucial that these profiles are of an appropriate size – not so small that they prevent us from showing meaningful information due to statistical disclosure rules, nor so large that they lose their usefulness. Our current sample size means some profiles are quite small, which can limit our ability to provide granular breakdowns, such as property type distributions within specific profiles. This issue is further enhanced by missing contextual information, given that a proportion of values is unknown for some variables, often requiring us to group unknown categories with known ones due to low counts.
+With a larger sample or enhanced synthetic data, we could further refine these profiles to identify ‘true archetypes’ of energy consumption and household characteristics. Making high-fidelity synthetic smart meter data openly available is crucial for leveraging this information to inform the energy transition.
```

Comment:
```suggestion
With a larger sample or enhanced synthetic data, we could further refine these profiles to identify ‘true archetypes’ of energy consumption and household characteristics.
```
I dont think the last sentence is required here because we're just talking about caveats.